src/main/java/org/fundacionjala/coding/dennis/Main.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacionjala.coding.dennis;
+
+/**
+ *
+ */
+final class Main {
+
+    public static final int THREE = 3;
+
+    /**
+     *
+     */
+    private Main() {
+    }
+
+    /**
+     *
+     * @param args Main.
+     */
+    public static void main(final String[] args) {
+        Customer customer = new Customer("Dennis");
+        customer.addRental(new Rental(new MovieNewRelease("First Man"), 2));
+        customer.addRental(new Rental(new MovieRegular("Frost Vs Nixon"), 2));
+        customer.addRental(new Rental(new MovieChildren("UP"), 2));
+        customer.addRental(new Rental(new MovieNewRelease("The favourite"), THREE));
+        customer.statement();
+        System.out.println("Bolsa tama�o: ".concat(Integer.toString(customer.capacityRentalElements())));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)